packages/embark-reset/index.js

```diff
@@ -0,0 +1,28 @@
+/* global exports process require */
+
+const {join} = require('path');
+const {promisify} = require('util');
+const rimraf = promisify(require('rimraf'));
+
+const dappPath = process.env.DAPP_PATH || process.cwd();
+
+const green = (text) => '\x1b[32m' + text + '\x1b[0m';
+
+exports.paths = new Set([
+  '.embark',
+  'chains.json',
+  'coverage',
+  'dist',
+  'embarkArtifacts',
+  'node_modules/.cache'
```

Comment:
I know you just moved those, but it would be great if the reset was actually smart and deleted the "right" directories based on config. 
Eg: if I change `embarkArtifacts` to be `src/myArtifacts`, this doesn't work

Reply:
I agree, but wasn't sure in this PR how best to make it more flexible. We could include a list of things in dapps' `embark.json` that should be deleted when running `embark reset`. That would be the most flexible, but has some downsides too I suppose.

I did already allow for the built-in list in `embark-reset/index.js` to be overriden.

Reply:
Ah ok, that's a good starting point